opencti-platform/opencti-graphql/src/modules/threatActorIndividual/threatActorIndividual.ts

```diff
@@ -76,15 +76,15 @@ const THREAT_ACTOR_INDIVIDUAL_DEFINITION: ModuleDefinition<StoreEntityThreatActo
       upsert: false,
       label: 'Threat actor types'
     },
-    { name: 'first_seen', type: 'date', mandatoryType: 'no', multiple: false, upsert: true },
-    { name: 'last_seen', type: 'date', mandatoryType: 'no', multiple: false, upsert: true },
+    { name: 'first_seen', type: 'date', mandatoryType: 'no', multiple: false, upsert: true, label: 'First seen' },
+    { name: 'last_seen', type: 'date', mandatoryType: 'no', multiple: false, upsert: true, label: 'Last seen' },
     { name: 'goals', type: 'string', mandatoryType: 'no', multiple: true, upsert: true },
```

Comment:
Why some have labels and others haven't ?

Reply:
It depends on localization management on the front side.
We use the name as key for localization but sometimes the key mismatch, so we use an other one already defined

Reply:
I see, thanks